tests/Event/EventTest.php

```diff
@@ -535,6 +535,267 @@ function (Event $event) use ($bookingInfo) {
+                    new PriceInfo(
+                        new BasePrice(
+                            new Money(
+                                100,
+                                new Currency('EUR')
+                            )
+                        )
+                    )
+                ),
+                new PriceInfoUpdated(
+                    'd2b41f1d-598c-46af-a3a5-10e373faa6fe',
+                    (new PriceInfo(new BasePrice(new Money(100, new Currency('EUR')))))
+                        ->withUiTPASTariffs([
+                            new \CultuurNet\UDB3\PriceInfo\Tariff(
+                                new MultilingualString(
+                                    new LegacyLanguage('nl'),
+                                    new StringLiteral('Tariff 1')
+                                ),
+                                new Money(
+                                    199,
+                                    new Currency('EUR')
+                                )
+                            ),
+                        ])
+                ),
+            ])
+            ->when(
+                fn (Event $event) => $event->updatePriceInfo(
+                    (new PriceInfo(
+                        new BasePrice(
+                            new Money(
+                                90,
+                                new Currency('EUR')
+                            )
+                        )
+                    ))->withUiTPASTariffs([
+                        new \CultuurNet\UDB3\PriceInfo\Tariff(
+                            new MultilingualString(
+                                new LegacyLanguage('nl'),
+                                new StringLiteral('Tariff 1')
+                            ),
+                            new Money(
+                                80,
+                                new Currency('EUR')
+                            )
+                        ),
+                    ])
+                )
+            )
+            ->then([
+                new PriceInfoUpdated(
+                    'd2b41f1d-598c-46af-a3a5-10e373faa6fe',
+                    (new PriceInfo(new BasePrice(new Money(90, new Currency('EUR')))))
+                        ->withUiTPASTariffs([
+                            new \CultuurNet\UDB3\PriceInfo\Tariff(
+                                new MultilingualString(
+                                    new LegacyLanguage('nl'),
+                                    new StringLiteral('Tariff 1')
+                                ),
+                                new Money(
+                                    199,
+                                    new Currency('EUR')
+                                )
+                            ),
+                        ])
+                ),
+            ]);
+    }
+
+    /**
+     * @test
+     */
+    public function it_ignores_an_update_with_equal_prices_on_uitpas_prices(): void
```

Comment:
The test name says `with_equal_prices_on_uitpas_prices` but the `updatePriceInfo()` call does not include uitpas prices, so they're not equal?

Reply:
I'll try to improve the names, I created tests for:
- update on Offer with different Base/Tariffs & No UiTPAS -> expects base & Tariffs update, UiTPAS should stay the same `it_keeps_existing_uitpas_prices_on_price_info_update()`
- update on Offer with different Base/Tariffs & different UitPAS -> expects only Base & Tariffs update, UiTPas should stay the same `it_ignores_an_update_of_uitpas_prices()`
- update an Offer without same Base Tariffs & no UiTPAS -> expects no Event recorded `it_ignores_an_update_with_equal_prices_without_uitpas()`
- update on Offer with same Base/Tariffs & different UiTPAS Prices -> expects no Event recorded `it_ignores_an_update_with_only_different_uitpas_prices()`

Reply:
Is the third point:

> update an Offer without same Base Tariffs & no UiTPAS -> expects no Event recorded `it_ignores_an_update_with_equal_prices_without_uitpas()`

intended to be:

> update an Offer **with** same Base Tariffs & no UiTPAS -> expects no Event recorded `it_ignores_an_update_with_equal_prices_without_uitpas()`

?

Reply:
Indeed, that was a confusing typo.

Reply:
Alrighty, aside from that it was clear to me! 👍